en/getting-started/first-game/index.md

```diff
@@ -370,173 +832,203 @@ export class GameManager extends Component {
+    It can be noticed that the engine automatically creates a Canvas node in the **Hierarchy** panel, a Player button node, and a Label node. That is because any node that contains a UI component only be seen when it is under the `Canvas` node, so the editor will create a Canvas in case there is none.
+
+    Change the **String** property of the `cc.Label` component in the Label Node from Button to Play.
 
-2. Create an empty node named `StartMenu` under `Canvas` and drag `PlayButton` under it. We can switch to the 2D editing view for UI editing operations by clicking the 2D/3D button on the toolbar.
+    > You may notice that the version of Cocos Creator you are using automatically adds a new Camera node when creating a UI node
+    > This node is a new feature in Cocos Creator. The new Camera will be in charge of the UI/2D parts, therefore the engine's render pipeline can process the UI/2D content more efficiently.
 
-    > **Note**: 2D View is this toolbar button ![2d-view](./images/2d-view.png).
+2. Create an empty node named StartMenu under the Canvas node, and drag the PlayerButton under it. We can switch to a 2D editing view for editing UI by clicking the 2D/3D button on the scene's toolbar. For more details on scene editing,  please refer to [Scene Panel](../../editor/scene/index.md)。
 
-    > **Note**: before proceeding, please read the [Scene Editing](../../editor/scene/index.md) documentation.
+    ![2d-view](./images/2d-view.png)
 
-3. Add a background frame by creating a `Sprite` node named `BG` under `StartMenu`. Adjust `BG`'s position to above the `PlayButton`, setting the __W(width)__ and __H(height)__ of `ContentSize` to __(200, 200)__, and setting its __SpriteFrame__ to `internal/default_ui/ default_sprite_splash`.
+3. Create a new Sprite node named `BG` under the `StartMenu` as a background image, and drag it above on the PlayButton.
 
     ![create bg sprite](./images/create-bg-sprite.gif)
 
+    Then set the `ContentSize` property of `cc.UITransform` component to (200, 200) on the **Inspector** panel, and drag the `internal/default_ui/default_sprite_splash` asset to its **SpriteFrame** property.
+
     ![change spriteFrame](./images/change-spriteFrame.png)
 
-4. Add a __Label__ called `Title` for the title of the start menu.
+4. Create a Label node named Title beneath the StartMenu node on the **Hierarchy** panel, and use it as the title of the start menu.
 
     ![add title label](./images/add-label-title.gif)
 
-5. Modify the text for `Title` and adjust it's *position*, *text size* and *color*.
+5. Adjust other properties of the Title node such as on `Position`, `Color`, `String`, `FontSize`, etc on the **Inspector** panel by the following image:
 
     ![modify title](./images/title-inspector.png)
 
-6. Adjust the position of the `PlayButton`. The layout of a simple __start menu__ is complete.
+6. Once the required Tips nodes are added and the PlayButton is adjusted, a simple start menu is complete.
```

Comment:
is complete -> is completed

both are correct, but we often use "is completed"  to emphasize that something have been finished just now.